tests/unit/ContactsTests.cpp

```diff
@@ -235,39 +235,39 @@ TEST(Contacts, ComputeContactsWithPolygons)
         {283.035034, 284.246307},
         {300.004883, 205.457642},
         {260.004578, 180.002838},
-        {meshkernel::doubleMissingValue, meshkernel::doubleMissingValue},
+        {meshkernel::constants::missing::doubleValue, meshkernel::constants::missing::doubleValue},
         {212.731567, 422.429504},
         {153.337280, 424.853821},
         {149.700867, 520.612366},
         {245.459045, 525.460876},
         {268.489502, 474.551270},
         {212.731567, 422.429504},
-        {meshkernel::doubleMissingValue, meshkernel::doubleMissingValue},
+        {meshkernel::constants::missing::doubleValue, meshkernel::constants::missing::doubleValue},
         {483.036316, 307.276855},
         {378.793213, 329.095245},
         {404.247925, 403.035400},
         {510.915283, 394.550476},
         {483.036316, 307.276855},
-        {meshkernel::doubleMissingValue, meshkernel::doubleMissingValue},
+        {meshkernel::constants::missing::doubleValue, meshkernel::constants::missing::doubleValue},
```

Comment:
using meshkernel::constants on the top of the .cpp file? so the namespace specification is shorter? maybe in the next pr

Reply:
Done.